<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ClickableView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Health</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Attack</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Defense</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Attack Range</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Move Range</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionText"/>
   <addaction name="actionMap"/>
   <addaction name="actionPlayers"/>
   <addaction name="actionStart"/>
   <addaction name="actionHeal"/>
  </widget>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
   <property name="toolTip">
    <string>Import Map</string>
   </property>
  </action>
  <action name="actionPlayers">
   <property name="text">
    <string>Players</string>
   </property>
   <property name="toolTip">
    <string>Import Players</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the game</string>
   </property>
  </action>
  <action name="actionHeal">
   <property name="text">
    <string>Heal</string>
   </property>
   <property name="toolTip">
    <string>Heal the Unit</string>
   </property>
  </action>
  <action name="actionText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Actions:</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableView</class>
   <extends>QGraphicsView</extends>
   <header>clickableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
